### PR TITLE
Package header files needed for IXrContext.h usage (OpenXR + UWP)

### PR DESCRIPTION
**Pull request by chrisfromwork**
This review enables developers to build custom openxr extensions for react-native-windows/IXrContext.

- [x] It requires the following pr/submodule update: https://github.com/BabylonJS/BabylonNative/pull/609

### CODE REVIEW

**Comment by chrisfromwork** [comment]
> 

this needs reverted once a commit exists in the BabylonJS repo for the needed changes

---
Refers to: .gitmodules:4 in e90c103. [](commit_id = e90c1037731977f39e9ff23ee3b5885bcc7d51b9, deletion_comment = False)

File: Package/gulpfile.js [hunk 1]
```diff
@@ -210,6 +213,27 @@ const copyVCXProjUWPFiles = () => {
     .pipe(gulp.dest('Assembled-Windows/windows/BabylonReactNative'));
 }
 
+const copyOpenXRInfoFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/LICENSE')
+  .pipe(gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Dependencies/xr/Dependencies/OpenXR-MixedReality/README.md'))
+  .pipe(gulp.dest('Assembled-Windows/windows/OpenXR-MixedReality'));
```
Comment (chrisfromwork) [inline Package/gulpfile.js:219]: @bghgary @ryantrem @tymlipari I am not sure if there are concerns around distributing headers from submodule dependencies. If this needs a separate npm package let me know :)

Comment (chrisfromwork) [reply Package/gulpfile.js:310]: Chatted with folks offline, headers are coming from another MIT license repo and should be fine.

**Review by rgerd — APPROVED** [review]

**Review by tymlipari — APPROVED** [review]

File: Package/gulpfile.js [hunk 2]
```diff
@@ -301,6 +304,32 @@ const copyVCXProjUWPFiles = () => {
...
+}
+
+const copyOpenXRPreviewHeaders = () => {
```
Comment (ryantrem) [inline Package/gulpfile.js:313]: Does it really make sense to have these as four separate functions, or should we just combine them into a single copyOpenXRHeaders that internally uses gulp.parallel?

Comment (chrisfromwork) [reply Package/gulpfile.js:313]: i have separate functions for files that copy to different directories. I'll go a head and combine copyOpenXRExtensionHeader and copyOpenXRHelperHeader

**Comment by chrisfromwork** [comment]
closing to reset build ci

**Pull request merged**